app/build.gradle

```diff
@@ -25,25 +25,22 @@ if (enable_plugin_kotlin) {
     apply plugin: 'kotlin-kapt'
 }
 
-ext.version_setup_minSdk = 14
```

Comment:
Defining a global default minSDK version for my projects, overriding project specific if needed. Thats whats done here, want to keep doing it that way :)